### DISCUSSION TITLE
Vitepress Sidebar Builder

### DISCUSSION THREAD

**Discussion by Stuyk** [vuejs/vitepress#1342] [Show and tell]
# Vitepress Sidebar Builder

Wrote a little plugin that helps you build a sidebar. There's some general configuration but it's fast and simple.

## Features

* Can read all markdown files in a folder.
  * Does not recurse.
* Can read all markdown files from a folder.
  * Recurses and finds all files from all folders.
* Orders files by an `order` front-matter property.
* Automatically uses `title` from `front-matter` property for sidebar title.
* Ability to ignore specific files from being captured.

Check it out here: https://github.com/stuyk/vitepress-sidebar-builder

## Example Output

![](https://i.imgur.com/TQgqNSF.png)

**Comment by isorna** [comment]
Thank you very much for sharing, I made my own version of this utility before finding it, luckily it'll help me improve mine hehe.
Regards
